spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin/ZipkinProperties.java

```diff
@@ -27,6 +27,7 @@
 	private String baseUrl = "http://localhost:9411/";
 	private boolean enabled = true;
 	private int flushInterval = 1;
+	private boolean gzip = false;
```

Comment:
per discussion, following convention of boot: `compression.enabled`, so.. `spring.zipkin.compression.enabled = true`